src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -64,6 +64,28 @@ public function get( WP_REST_Request $request ) {
 			: false;
 		$args['s'] = $request['search'];
 
+		/**
+		 * Allows users to make the REST API return events from a more specific, localized date range.
```

Comment:
Hey don't let this remark hold anything up ... _but (!)_ ... is it a more specific and localized range or merely a different range?

Reply:
..should we quickly detail how it works if this were 'false' (ie, the behaviour prior to the introduction of this filter and the new default)?

Reply:
I like that improvement

Reply:
@barryhughes no worries, i've not made changes on the REST API so want to get this right _(not i don't want to get everything right, but you know what i mean...)_

as an example, lets say the REST url  looks like this:
```
http://tribe.dev/wp-json/tribe/events/v1/events?start_date=2017-12-21&end_date=2017-12-22
```
if your site's in the `America/New_York` timezone, then these are the "specific and localized" dates ultimately passed to `tribe_get_events()` call in the GET request:
```
    [start_date] => 2017-12-20 19:00:00
    [end_date] => 2017-12-21 19:00:00
```
☝️ that's how it was before my fix here

now it has new behavior (which u can disable with the filter if u want) that would make it so that with that same REST request url, _these_ values — not localized, not "specific" because it's just "beginning of day" and "end of day", get passed to `tribe_get_events()`:

```
[start_date] => 2017-12-21 00:00:00
[end_date] => 2017-12-22 23:59:59
```
_(assuming a midnight 'end of day cutoff' setting...adjusts based on that)_

Reply:
@bordoni @barryhughes lol, i was typing for long enough that once i posted my comment i saw your two new ones.

yes, i could add a more compressed version of my comments here to that docblock. is that what you mean?

Reply:
Yep pretty much - a nice clear example like you provided there showing the behaviour when true and when false.